pkg/virtctl/hotplug/common.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 The KubeVirt Contributors
+ *
+ */
+
+package hotplug
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const RequestTimeout = 10 * time.Second
+
+func ValidateVirtualMachineExists(ctx context.Context, virtCli kubecli.KubevirtClient, namespace, name string) error {
+	_, err := virtCli.VirtualMachine(namespace).Get(ctx, name, k8smetav1.GetOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf("resource not found: %s/%s", namespace, name)
```

Comment:
What do you gain by wrapping an `IsNotFound` error in another error?

Reply:
nothing it's leftovers I forgot to clean, fixed